eclass/qt5-build.eclass

```diff
@@ -657,7 +658,11 @@ qt5_base_configure() {
 
 	# a forwarding header is no longer created since 5.8, causing the system
 	# config to always be used. bug 599636
-	cp src/corelib/global/qconfig.h include/QtCore/ || die
+	local basedir="${S}/"
+	if ver_test ${PV} -lt 5.14.2 || [[ ${QT5_BUILD_TYPE} == live ]]; then
+		basedir=""
+	fi
+	cp src/corelib/global/qconfig.h "${basedir}"include/QtCore/ || die
```

Comment:
we shouldn't touch `$S` if we're doing an out-of-source build... OTOH it doesn't really matter at the moment. Does it really not work without `$S`?

Reply:
since live and release behaviour differ, the other option would have been creaiting a faux .git directory in $S. but I don't know what other changes that might bring.

Reply:
no I wouldn't do that. This whole thing is a giant hack anyway.

Reply:
See `.git` checks:
https://code.qt.io/cgit/qt/qtbase.git/tree/configure?h=5.15&id=6387138a7991b4588639dc48847f175b5afaff84#n692
https://code.qt.io/cgit/qt/qtbase.git/tree/configure?h=5.15&id=6387138a7991b4588639dc48847f175b5afaff84#n752

Reply:
@Arfrever what are you trying to say exactly?